src/release-notes/v2022.01.28-0.md

```diff
@@ -0,0 +1,12 @@
+---
+date: "2022-01-28"
+version: "v2022.01.28-0"
+weight: 202201280
+---
+
+### <span class="label label-green">New Features</span>
+- Registry backup and restore scripts includes more user-friendly logging within the container.
+- Add support for Kubernetes 1.23.x
+
```

Comment:
```suggestion
- Add support for Kubernetes 1.23.x
- Registry backup and restore scripts include more user-friendly logging within the container.

```

because one of these is a lot more important than the other 😆

Reply:
(and also I removed a 's' to fix grammar)